nomad-aws/examples/basic/main.tf

```diff
@@ -51,6 +51,9 @@ module "nomad-aws" {
     module.vpc.private_subnets[0]
   ]
 
+  # cidr block you’d like to use in docker within nomad client, should not be same as subnet cidr block
+  docker_network_cidr = "192.168.0.0/16"
```

Comment:
My comment from the [sibling PR](https://github.com/CircleCI-Public/server-terraform/pull/146#pullrequestreview-957875537) stands here as well.  Lets not include this in the basic example